src/main/java/org/openRealmOfStars/player/ship/ShipComponentType.java

```diff
@@ -455,6 +468,7 @@ public int getHitChance() {
           return 75;
       case WEAPON_ECM_TORPEDO:
       case WEAPON_HE_MISSILE:
+      case CALLISTO_MULTICANNON:
```

Comment:
I was thinking that multicannon would be between missile and railgun by accuracy. So maybe 60 would be good for this. Also other weapons get range update on tech level 6 and 9. Some upgrade would be required also for multicannon. Could that be maybe increase in hit accuracy? In order to do so there would be need for different type for levels 6 and 9.